netavark.spec.rpkg

```diff
@@ -58,12 +60,25 @@ popd
 %install
 %{__make} DESTDIR=%{buildroot} PREFIX=%{_prefix} install
 
+%post
+if [ $1 -eq 1 ]; then
+    %{_bindir}/systemctl enable --now %{name}-dhcp-proxy.service
+fi
```

Comment:
I still do not think we should enable it, also if we need would want to enable by default it should enable the socket not service.